web/documentserver-example/ruby/app/controllers/home_controller.rb

```diff
@@ -157,6 +157,28 @@ def convert
 
   end
 
+  def historyobj
+    begin
+      data = request.body.read
+      if data == nil || data.empty?
+          return ""
+      end
+      fileData = JSON.parse(data)
+      file = FileModel.new(
+        :file_name => File.basename(fileData['file_name']), 
+        :mode => fileData['mode'], 
+        :type => fileData['type'], 
+        :user_ip => fileData['user_ip'], 
+        :lang => fileData['lang'], 
+        :user => fileData['user'], 
+        :action_data => fileData['action_data'], 
+        :direct_url => fileData['direct_url']
+      )
+      history = file.get_history
+      render json: history
+    end
```

Comment:
You don't need to use a `begin-end` block here. Just put the code directly into the method without wrapping it in a `begin-end` block. Or you can add error handling using the `rescue` keyword.